packages/uicore/src/components/Select/Select.tsx

```diff
@@ -61,6 +61,7 @@ export interface SelectProps
   onQueryChange?: Props['onQueryChange']
   addTooltip?: boolean
   borderless?: boolean
+  loadingItems?: boolean
```

Comment:
Can we add a comment for this? This will avoid confusion for anybody trying to add a loading prop to the select component

Reply:
Ignore